rskj-core/src/test/java/co/rsk/core/bc/TransactionPoolImplTest.java

```diff
@@ -223,23 +215,8 @@ public void addAndExecutePendingTransaction() {
 
         transactionPool.addTransaction(tx);
 
-        Repository repository = transactionPool.getRepository();
-        Assert.assertEquals(BigInteger.valueOf(1001000), repository.getBalance(receiver.getAddress()).asBigInteger());
-    }
-
-    @Test
```

Comment:
I would try to keep this test reworked

Reply:
This very same test is implemented bellow in the `updateTransactionPool` method